feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -388,7 +395,9 @@ func (tc *testCase) testFlow(t *testing.T, packetSize uint16, ipHeader ondatra.H
 	} else if avg := octets / inPkts; avg > uint64(tc.mtu) {
 		t.Errorf("Flow destination packet size average got %d, want <= %d (MTU)", avg, tc.mtu)
 	}
-	if inPkts < p2OutDiff.unicast {
+
+	// Tolerance added for ipv6 control packets
+	if inPkts+Ipv6ControlPacketTolerance < p2OutDiff.unicast {
```

Comment:
Instead of adding a tolerance, can we check the unicast counters instead if they are available?

Reply:
the ipv6 advertisement message are unicast. the issue is that ixia does not account for control packets  for flow counters.

Reply:
NS message is a solicit multicast, a NA message is a unicast. So, the exchange is just 50% unicast